versions/sles_15.2.ymlx

```diff
@@ -0,0 +1,5 @@
+osDistro: sles
+osVersion: 15.2
+packages:
+  - arch: x86_64
```

Comment:
Can we get this confirmed before publishing to stage as well

Reply:
we are not including 15.2 sles for now . should look into it later finding the right ami.

Reply:
If infra agent still supports it then which artifact they would get if they are on this OS Distro?

Reply:
they will get the previous fluent bit version which supports this OS Distro , in this case that could be 3.1.2